config/routes.rb

```diff
@@ -13,6 +13,8 @@
   post 'signup' => 'user_registration#create'
   post 'logout' => 'user_sessions#destroy', :as => :logout
 
+  get 'decks' => 'decks#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.